src/utils.py

```diff
@@ -0,0 +1,7 @@
+SCRAPER_INFO_FILE_NAME = "apps"
```

Comment:
Optional: if you want to make the code a bit more "google-style" you could use absl.flags to set these file names up in a configurable way.

Reply:
Not necessary in this PR, but i would suggest to discuss with Seb some more elegant way to handle these config variables, storing them in a python file like this seems odd to me.